packages/astro/src/default/utils/content.ts

```diff
@@ -10,6 +10,7 @@ import type {
 import { folderPathToFilesRef, interpolateString } from '@tutorialkit/types';
 import { getCollection } from 'astro:content';
 import glob from 'fast-glob';
+import mm from 'micromatch';
```

Comment:
Super Nit pick: I would either import `isMatch` or call the import `micromatch`.

Reply:
Sure! This is how we've used it in Vitest: https://github.com/search?q=repo%3Avitest-dev%2Fvitest%20micromatch&type=code
Though it looks like in Vite it's imported as `micromatch`. The package is CJS and doesn't seem to work with named imports.